third_party/move/move-bytecode-verifier/transactional-tests/tests/stack_usage_verifier/signer_move_to_extra_arg.exp

```diff
@@ -11,9 +11,9 @@ Error: Unable to publish module '00000000000000000000000000000000000000000000000
 
 task 1 'publish'. lines 15-24:
 Error: Unable to publish module '0000000000000000000000000000000000000000000000000000000000000042::N'. Got VMError: {
-    major_status: POSITIVE_STACK_SIZE_AT_BLOCK_END,
+    major_status: CONSTRAINT_NOT_SATISFIED,
```

Comment:
Is this a breaking change (covered by the feature flag)?

Reply:
Yes it is a breaking change. The whole pass is now feature-gated.

Reply:
Actually let me double check. This one is changing from `POSITIVE_STACK_SIZE_AT_BLOCK_END` to `CONSTRAINT_NOT_SATISFIED`, which is not expected. Either the test or the new sig checker needs to be fixed.